docs/nydus-image.md

```diff
@@ -1,65 +1,187 @@
+- a bootstrap/meta blob, containing filesystem metadata, such directory name, file attributes etc.
+- one or more data blobs, containing file contents.
 
-### Get binary from release page
+Physically, RAFS bootstrap/meta blobs can be stored as separate files, or inlined into data blob files.
+Therefore, a RAFS file system may consist of the following parts:
+- a blob file containing both RAFS metadata and data
+- a blob file containing RAFS metadata, and one or more blob files for RAFS data
+- a blob file containing RAFS metadata, and one or more blob files for RAFS data, and associated targz files for RAFS ZRAN mode.
 
-Get `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
+## Installation
 
-## Build Nydus Image From Directory Source
+Get latest `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
+## Nydus Image Builder
+
+The `nydus-image create` subcommand creates a RAFS filesystem/Nydus image from a tar file or from a directory.
+
+RAFS filesystem/Nydus image has three modes: Native, Zran and Tarfs. Each mode has different features and may be used for different scenarios.
+
+|  Mode  |    Blobs in Registry    |       Local Cache File        | Runtime Integrity | Lazy-loading | Chunk Dedup | Encryption | OCIv1 Compatible |
+|:------:|:-----------------------:|:-----------------------------:|:-----------------:|:------------:|:-----------:|:----------:|:----------------:|
+| Tarfs  |    tar.gz / tar.zst     |       nydus.meta & tar        |     Optional      |      No      |         No          |     No     |       Yes        |
+| Zran   |   tar.gz & nydus.meta   | nydus.meta & nydus.data.cache |        Yes        |     Yes      |      Yes       |     No     |       Yes        |
+| Native | nydus.data & nydus.meta | nydus.meta & nydus.data.cache |       Yes         |      Yes     |     Yes      |    Yes     |        No        | 
+
+### Specify Data Blob Output Path
+
+There are two ways to specify where to save the resulting data blob:
+
+- Specify the file path via `--blob <BLOB_FILE>`. It could be a regular file into which the data blob contents are dumped. It can also be a fifo (named pipe) from which "nydusify" or other tools can receive the generated blob content.
+
+- Specify a directory with `-D/--blob-dir BLOB_DIR`. `nydus-image` will use the sha256 digest of the resulting data blob as the filename, concatenated to the directory path. This is useful when you don't want to set a custom name or you are building a layered nydus image. Please create `BLOB_DIR` before executing the command.
+
+### Build RAFS Filesystem in Native Mode from a Directory
 ```shell
-nydus-image create \
-  --bootstrap /path/to/bootstrap \
-  --blob /path/to/blob \
+nydus-image create -t dir-rafs \
+  -D /path/to/output/directory \
   /path/to/source/dir
+  
+[root@image-service]# target/debug/nydus-image create -t dir-rafs -D images/ src 
```

Comment:
It's better to remove the `target/debug/` prefix for all `nydus-image` commands.

Reply:
fixed